sphinx_gallery/gen_rst.py

```diff
@@ -585,8 +580,7 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
                           for blabel, bcontent in script_blocks
                           if blabel == 'code'])
 
-    first_image_path = image_path_template.format(1)
-    if _plots_are_current(example_file, first_image_path):
+    if md5sum_is_current(example_file):
         return amount_of_code, 0
```

Comment:
which other branches? sure you are comparing this function and not the execute_script?